src/applications/ivc-champva/10-10D/pages/ApplicantRelationshipPage.jsx

```diff
@@ -78,15 +103,16 @@ const relationshipStructure = {
 };
 
 export function ApplicantRelationshipReviewPage(props) {
-  const { data } = props || {};
+  const { data, keyname = KEYNAME } = props || {};
+  const genOps = props.genOp || generateOptions;
```

Comment:
This lets us target a different property on the applicant (i.e., relationship to sponsor vs relationship origin) and use a custom `generateOptions` function to produce the right custom radio button text while falling back to the original property and generate function if needed.